docs/how-tos/entanglement_forging/use-asymmetric-bitstrings.ipynb

```diff
@@ -154,15 +139,13 @@
    "outputs": [],
    "source": [
     "# Create an ElectronicStructureProblem from our IntegralDriver and performing second quantization transformation\n",
-    "driver = IntegralDriver(\n",
-    "    hcore=one_body_integrals_alpha,\n",
-    "    mo_coeff=np.eye(6, 6),\n",
-    "    eri=two_body_integrals_alpha_alpha,\n",
-    "    num_alpha=3,\n",
-    "    num_beta=2,\n",
-    "    nuclear_repulsion_energy=nuclear_repulsion_energy,\n",
+    "hamiltonian = ElectronicEnergy.from_raw_integrals(\n",
+    "    one_body_integrals_alpha, two_body_integrals_alpha_alpha\n",
     ")\n",
-    "problem = ElectronicStructureProblem(driver)"
+    "hamiltonian.nuclear_repulsion_energy = nuclear_repulsion_energy\n",
+    "problem = ElectronicStructureProblem(hamiltonian)\n",
+    "problem.num_particles = (3, 2)\n",
+    "problem.basis = ElectronicBasis.MO"
```

Comment:
Why is it necessary to build this out again? Is this not the same information as contained in `problem_reduced`?

EDIT: could it be that you are doing this in order to strip out the beta-spin information?

Reply:
Yes, Ieva wrote this, but I believe she strips out some spin information first and then builds a new problem from that